### ISSUE TITLE
Check metadata for all types in hierarchy

### ISSUE THREAD

**Issue by vladsch** [bridgedotnet/Bridge.Newtonsoft.Json#2]
By default, only top level class metadata is checked before serialization but need to check all base types

## See Also

* https://forums.bridge.net/forum/bridge-net-pro/bugs/4371-reflectable-check-not-applied-to-base-classes

**Comment by geoffreymcgill** [comment]
A pull request has been submitted with an implementation of this feature. Once the pull request has been reviewed and testing is complete, we will merge into the **master** branch and include with the next release.
